jboss/container/wildfly/launch/datasources/added/launch/datasource-common.sh

```diff
@@ -40,6 +40,8 @@ function clearDatasourceEnv() {
   unset ${prefix}_TX_ISOLATION
   unset ${prefix}_MIN_POOL_SIZE
   unset ${prefix}_MAX_POOL_SIZE
+  unset ${prefix}_IS_SAME_RM_OVERRIDE
```

Comment:
It would be nice if you could add these 2 new env to the https://github.com/wildfly/wildfly-cekit-modules/blob/master/jboss/container/wildfly/launch/datasources/module.yaml file in order to document new env vars.